src/App.tsx

```diff
@@ -1,24 +1,332 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable no-console */
+import React, {
+  useEffect, useState, useMemo, useRef,
+} from 'react';
+import { Todo, SortType } from './Types';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './todos';
+import { client } from './utils/client';
+import { Error } from './components/ErrorMessages';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
 
-const USER_ID = 0;
+const USER_ID = 10377;
+
+function getRandomNumber(): number {
```

Comment:
Use an external library like 'uuid' or 'nanoid' to generate unique ids to avoid possible collisions instead of using getRandomNumber().